packages/components/src/button/test/index.tsx

```diff
@@ -402,6 +402,11 @@ describe( 'Button', () => {
 			);
 			expect( console ).toHaveWarned();
 		} );
+
+		it( 'should not break when the legacy isSmall prop is passed', () => {
```

Comment:
Should we also add a test about the interaction between `isSmall` and `size`, to make sure that they behave as expected in terms of which prop has priority?

Reply:
Good idea 9ec488a